docs/MO_DG/prepare_model/customize_model_optimizer/Customize_Model_Optimizer.md

```diff
@@ -6,217 +6,218 @@
    :maxdepth: 1
    :hidden:
 
```

Comment:
Now you are adding `Model Optimizer Extensibility` for legacy into `OpenVINO Extensibility` part. In `OpenVINO Extensibility` we should have only extensibility logic for new frontends.